core/src/test/kotlin/com/tschuchort/compiletesting/KotlinCompilationTests.kt

```diff
@@ -859,6 +859,50 @@ class KotlinCompilationTests {
 		)
 		fakeJdkHome.delete()
 	}
-	
+
+	@Test
+	fun `the classLoader from a 2nd compilation can load classes from the first compilation`() {
+		val kSource1 = SourceFile.kotlin(
+			"KSource1.kt", """
+				package com.tschuchort.compiletesting
+			
+				interface KSource1
+				"""
+		)
+
+        val result = defaultCompilerConfig()
+            .apply {
+                sources = listOf(kSource1)
+                inheritClassPath = true
+            }
+            .compile()
```

Comment:
I think you're mixing tabs and spaces here.

Reply:
Resolved